examples/demo/public/pages/json.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'preact/hooks';
+import json from './foo.json';
+
+export function JSONView() {
+	const [fetched, setFetched] = useState(null);
+
+	useEffect(() => {
+		fetch('./pages/foo.json')
```

Comment:
Just a thought: this feels like a missed opportunity to demonstrate `import jsonUrl from 'url:./foo.json';` + `fetch(jsonUrl)`. Otherwise, to the reader the relative path `./pages/foo.json` looks like "magic", in the base context of `public/pages/json.js`.